src/tests/stdlib/replace_regex_ext.ab

```diff
@@ -0,0 +1,8 @@
+import * from "std/text"
+
+// Output
+// abc[123]def
+
+main {
+    echo replace_regex("abc123def", "([0-9]+)", "[\1]", true)
```

Comment:
This test would fail on a machine that does not support the new flag, right? Can we at least write a note somewhere or maybe even skip this test?

Reply:
I would prefer to keep the test, because it confirms that on systems which do support `-E` or `-r`, the right option is being used.  And while Amber may be run on older systems, I doubt that anyone will run `cargo test` on them.  Would you be happy with a comment in the test file?

Reply:
> I think that we should create some syntax for interfacing commands given different versions and os's

Makes sense, would need to give it some thought, but happy to discuss.  In the meantime, I've added this warning comment.  Does this solve your immediate request?
```
// This will fail on any system where sed does not support extended
// regex syntax, via "-r" on GNU sed and "-E" on all other versions.
```